components/nexus-core/src/main/java/org/sonatype/nexus/internal/orient/DatabaseServerImpl.java

```diff
@@ -129,7 +130,7 @@ protected void doStart() throws Exception {
     this.server = server;
   }
 
-  private OServerConfiguration createConfiguration() {
+  private OServerConfiguration createConfiguration() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)